docs/support_files/attachments.md

```diff
@@ -81,3 +81,20 @@ After(function (testCase) {
   }
 });
 ```
+
+Attachments are also printed by the progress, progress-bar and summary formatter.
+They appears right after the step and only `text/plain` content is visible.
+It can be used to debug scenarios, especially in parallel mode.
+
+```
```

Comment:
Can we please use javascript syntax highlighting here? The result format could be entirely commented out or put in its own block